security/certManager/file/certManager.go

```diff
@@ -14,23 +14,25 @@ import (
 
 // Config provides configuration of a file based Certificate manager
 type Config struct {
-	CAPool          string `envconfig:"CA_POOL" long:"tls-file_ca-pool" description:"file path to the root certificate in PEM format"`
-	TLSKeyFileName  string `envconfig:"CERT_KEY_NAME" long:"tls-file-cert-key-name" description:"file name of private key in PEM format"`
-	DirPath         string `envconfig:"CERT_DIR_PATH" long:"tls-file-cert-dir-path" description:"dir path where cert/key pair are saved"`
-	TLSCertFileName string `envconfig:"CERT_NAME" long:"tls-file-cert-name" description:"file name of certificate in PEM format"`
+	CAPool                       string `envconfig:"CA_POOL" long:"tls-file_ca-pool" description:"file path to the root certificate in PEM format"`
+	TLSKeyFileName               string `envconfig:"CERT_KEY_NAME" long:"tls-file-cert-key-name" description:"file name of private key in PEM format"`
+	DirPath                      string `envconfig:"CERT_DIR_PATH" long:"tls-file-cert-dir-path" description:"dir path where cert/key pair are saved"`
+	TLSCertFileName              string `envconfig:"CERT_NAME" long:"tls-file-cert-name" description:"file name of certificate in PEM format"`
+	DisableVerifyPeerCertificate bool   `envconfig:"DISABLE_VERIFY_PEER_CERTIFICATE" env:"DISABLE_VERIFY_PEER_CERTIFICATE" long:"disable-verify-peer-certificate" default:"true" description:"disable verify peer ceritificate"`
```

Comment:
This cfg property: **DisableVerifyPeerCertificate** is true by default. That means that if you don't change configuration parameters, you disable ClientCert verification by default. Is that correct behavior ?

Reply:
No. By default it must be false.